app/services/mailer/mailer.js

```diff
@@ -0,0 +1,56 @@
+const nodemailer = require('nodemailer');
+const hbs = require('nodemailer-express-handlebars');
+
+async function sendConfirmationEmail(toEmail) {
+  // create reusable transporter object
+  let transporter = nodemailer.createTransport({
```

Comment:
can we create a transport only one time to use for the express server instead of creating each time?